semantic-conventions/src/opentelemetry/semconv/main.py

```diff
@@ -223,6 +244,12 @@ def setup_parser():
     parser.add_argument(
         "--exclude", "-e", help="Exclude the matching files using GLOB syntax", type=str
     )
+    parser.add_argument(
+        "--reference-only",
+        "-r",
+        help="Additional files to resolve references only (and extends) using GLOB syntax",
```

Comment:
I think this feature warrants a bit more description. This suggestion is also a check if I understood this PR correctly 😃 

```suggestion
        help="""Additional files that will be loaded and available for referencing, but will not be part of the output.
        
        As semantic conventions can, in various ways, e.g. using `ref` or `include`, reference other conventions you
        sometimes need to add files/groups to your input but you don't want the groups in there be processed by
        the code generator template or markdown generator. In this case, add only the subset of files you want to
        process with your template / markdown generator to `--yaml-root`, and add any additional required
        (transitively depended-upon) files with `--reference-only`.
        """,
```